embedded-cassandra/embedded-cassandra-war/pom.xml

```diff
@@ -26,41 +26,50 @@
     <version>0.6.1.Final-SNAPSHOT</version>
   </parent>
 
-  <artifactId>hawkular-commons-embedded-cassandra-ear</artifactId>
-  <packaging>ear</packaging>
```

Comment:
I couldn't find out why this was previously packaged as EAR. And why EJB's had to be removed in order to change the packaging from EAR to WAR... Could you clarify that?